bin/deploy-release.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+set -eo pipefail
```

Comment:
These two lines are partially redundant. Just use a bare `bash` and `set -exo pipefail`